content/news/core-monthly/2024/core-monthly-2024-02-01-2024-02-29.md

```diff
@@ -0,0 +1,409 @@
+
+![Core Monthly banner](/assets/images/theme/banner-core-monthly-wide.jpg)
+
+### Release of PrestaShop 8.1.4
+
+The 8.1.4 maintenance release of PrestaShop was a dedicated security release. This was the first instance where the decision was made to separate security fixes from other bug fixes and improvements, allowing for a quicker release of the security patch. The release went smoothly, leading to plans to continue this approach in future releases. It is recommended to upgrade to this version promptly to benefit from these security fixes. More details can be found in the [release notes]({{< ref "/news/releases/prestashop-8-1-4-maintenance-release.md" >}}).
+
+### PrestaShop 8.1.5
+
+The work on the next maintenance release, 8.1.5, has already started. The release is planned for March and will include a number of bug fixes and improvements. I'll share more on this in a dedicated blog post soon.
+
+### PrestaShop 9.0
+
+**PHP 8.3 compatibility**
+
+PHP 8.3 [has been enabled](https://github.com/PrestaShop/PrestaShop/pull/34485) for the `develop` branch on GitHub. [GitHub workflows in the project](https://github.com/PrestaShop/PrestaShop/actions/workflows/php.yml) now runs on PHP 8.1, 8.2 and 8.3, and all the tests are passing. This is a significant step towards ensuring that PrestaShop 9.0 will be compatible with the latest version of PHP.
+
+**New API**
+
+The progress of the new API for PrestaShop 9.0 is going well. This new API is being built using the API Platform framework, which gives a strong base for creating modern web APIs. It is a significant upgrade from the current one, providing developers with a more consistent and predictable experience.
+
+One of the recent additions to the project is [introduction of `MutationTracker`](https://github.com/PrestaShop/PrestaShop/pull/35292), which is responsible for tracking some of the changes made to the entities. Thanks to this, the API will be able to provide more information about the changes made to the resources. One example of this is the ability to track modifications to orders history via API calls. What is important to underline is that this is not a logging system, but a way to track changes made to the resources.
+
+It's worth mentioning that the API Platform [has been updated to version 3.2](https://github.com/PrestaShop/PrestaShop/pull/35266). We wanted to make sure that we are using the latest version of the API Platform framework, which will provide us with the latest features, improvements and fixes.
+
+In the upcoming weeks, we will be releasing a series of blog posts to introduce the new API and its features. Stay tuned for more details.
+
+**Symfony 6**
+
+The work on upgrading the Symfony framework to version 6 is ongoing. The upgrade is a significant task, as it requires changes to many parts of the codebase. The upgrade will bring many improvements and new features, and it will also ensure that PrestaShop is using the LTS version of the the Symfony framework.
```

Comment:
```suggestion
The work on upgrading the Symfony framework to version 6 is ongoing. The upgrade is a significant task, as it requires changes to many parts of the codebase. The upgrade will bring many improvements and new features, and it will also ensure that PrestaShop is using the LTS version of the Symfony framework.
```